app/Notifications/Banking/AuditResult.php

```diff
@@ -72,7 +75,12 @@ public function __construct(
      */
     public function via($notifiable)
     {
-        return ['mail', 'slack'];
+        $channels = ['mail', 'slack'];
+        if (config('services.discord.token')) {
+            array_push($channels, DiscordChannel::class);
```

Comment:
🤢old syntax 

```suggestion
            $channels[] = DiscordChannel::class;
```

Reply:
:D what happens when you haven't done much php since 2010

Reply:
I've pretty much been a full time Laravel developer since the start of lock down